flake.nix

```diff
@@ -109,22 +109,7 @@
           let
             inherit (arch2targets.${system}) rustTarget;
           in
-          (craneLib.buildPackage (
-            (generateCommonArgs craneLib)
-            // {
-              postInstall = ''
-                cd "$out"/bin
-                for f in "$(ls)"; do
-                  if ext="$(echo "$f" | grep -oP '\.[a-z]+$')"; then
-                    base="$(echo "$f" | cut -d. -f1)"
-                    mv "$f" "$base-${rustTarget}$ext"
-                  else
-                    mv "$f" "$f-${rustTarget}"
-                  fi
-                done
-              '';
-            }
-          ));
```

Comment:
This is used to generate the file for the github release

<img width="539" height="139" alt="Image" src="https://github.com/user-attachments/assets/31f3671a-667d-4420-8f97-e20242277f74" />

So, happy to keep the system package as not renamed, but the files with the targets are still required